src/vcpkg/vcpkgcmdarguments.cpp

```diff
@@ -691,11 +693,26 @@ namespace vcpkg
 
     void VcpkgCmdArguments::track_environment_metrics() const
     {
+        MetricsSubmission submission;
         if (auto ci_env = m_detected_ci_environment.get())
         {
             Debug::println("Detected CI environment: ", *ci_env);
             get_global_metrics_collector().track_string(StringMetric::DetectedCiEnvironment, *ci_env);
         }
+
+        if (auto repo_id = github_repository_id.get())
+        {
+            Debug::println("Github Repository Id: ", *repo_id);
+            submission.track_string(StringMetric::GithubRepositoryId, *repo_id);
+        }
+
+        if (auto owner_id = github_repository_owner_id.get())
+        {
+            Debug::println("Github Repository Owner Id: ", *owner_id);
+            submission.track_string(StringMetric::GithubRepositoryOwnerId, *owner_id);
+        }
```

Comment:
```suggestion
        if (auto repo_id = github_repository_id.get())
        {
            submission.track_string(StringMetric::GithubRepositoryId, *repo_id);
        }

        if (auto owner_id = github_repository_owner_id.get())
        {
            submission.track_string(StringMetric::GithubRepositoryOwnerId, *owner_id);
        }
```

Let's avoid cluttering the output, we can use `--printmetrics` to debug.
This debug message could lead to PII being submitted when we ask for full logs.